### DISCUSSION TITLE
Sugerencia: casilla del día de nacimiento

### DISCUSSION THREAD

**Discussion by eggbanana173** [nesbox/TIC-80#1812] [General]
ya sabemos que TIC 80 es una "computadora fantasía" y tiene muchos juegos, pero también hay juegos no aptos para todo publico
(ósea +18)
por esto va mi sugerencia, tener una casilla del día de nacimiento cuando creas una cuenta para saber si eres mayor de edad, y si lo eres, poder acceder a esos juegos

**Comment by darltrash** [comment]
Translation:
# Birthday selection screen:
We already know that Tic80 is a "fantasy computer" and it has a lot of games, but there are also games that are not suitable for all ages (as an example, mature games)
So here's my suggestion: Adding a birthday selector at the account creation process to know if you're able to access the aforementioned games.

**Comment by nesbox** [comment]
But you don't need an account to play such games.
